utility_analysis_new/tests/dp_engine_test.py

```diff
@@ -183,7 +211,8 @@ def test_aggregate_public_partition_e2e(self):
         self.assertLen(col, 3)
         self.assertTrue(any(v[0] == 'pk101' for v in col))
 
-    def test_aggregate_error_metrics(self):
+    @parameterized.parameters(False, True)
+    def test_aggregate_error_metrics(self, pre_aggregated: bool):
```

Comment:
nit: It's a bit difficult to guess what's being tested by looking at the test name.